index-test.js

```diff
@@ -449,6 +453,12 @@ describe(`reactElementToJSXString(ReactElement)`, () => {
     ).toEqual(`<TESTCOMPONENT />`);
   });
 
+  it('reactElementToJSXString(<DefaultPropsComponen />)', () => {
+    expect(
+      reactElementToJSXString(<DefaultPropsComponent />)
```

Comment:
> Removes default props if their value is the default.

I am not sure to follow what you are trying to prove in this test, shouldn't we use test='test' somewhere?